docs/send-data/kubernetes/logs.md

```diff
@@ -0,0 +1,553 @@
+across Pod restarts and buffering in case of exporting problems.
+
+This feature is enabled by default, but it only works if you have a correctly configured default `storageClass` in your cluster. Cloud
+providers will do this for you when provisioning the cluster. The only alternative is disabling persistence altogether.
+
+Persistence can be customized via the `metadata.logs.persistence` section:
+
+```yaml
+metadata:
+  persistence:
+    enabled: true
+    # storageClass: ""
+    accessMode: ReadWriteOnce
+    size: 10Gi
+    ## Add custom labels to all otelcol statefulset PVC (logs and metrics)
+    pvcLabels: {}
+```
+
+:::note
+These settings affect persistence for metrics as well.
+:::
+
+## Advanced Configuration
+
+This section covers more advanced ways of configuring logging. Knowledge of OpenTelemetry Collector configuration format and concepts will
+be required.
+
+### Direct configuration
+
+There are two ways of directly configuring OpenTelemetry Collector for both log collection and metadata enrichment. These are both advanced
+features requiring a good understanding of this chart's architecture and OpenTelemetry Collector configuration.
+
+The `metadata.logs.config.merge` and `otellogs.config.merge` keys can be used to provide configuration that will be merged with the Helm
+Chart's default configuration. It should be noted that this field is not subject to normal backwards compatibility guarantees, the default
+configuration can change even in minor versions while preserving the same end-to-end behaviour. Use of this field is discouraged - ideally
+the necessary customizations should be able to be achieved without touching the otel configuration directly. Please open an issue if your
+use case requires the use of this field.
+
+The `metadata.logs.config.override` and `otellogs.config.override` keys can be used to provide configuration that will be completely replace
+the default configuration. As above, care must be taken not to depend on implementation details that may change between minor releases of
+this Chart.
+
+See [Sumologic OpenTelemetry Collector configuration][configuration] for more information.
+
+### Disabling container logs
+
+Container logs are collected by default. This can be disabled by setting:
+
+```yaml
+sumologic:
+  logs:
+    container:
+      enabled: false
+```
+
+### Multiline unstructured logs with HTTP sources
+
+By default, the Helm Chart sends data to Sumo using the OpenTelemetry Protocol (OTLP), and therefore uses the [OTLP Source][otlp_source].
+However, if you've chosen to use a [plain Http Source][http_source] by setting `sumologic.logs.source_type` to `http`, be aware that
+this source does not support client-side multiline parsing for logs in `text` format. You'll need to do multiline detection in the source
+itself. This can be set up in the Helm Chart configuration the following way:
```

Comment:
```suggestion
By default, the Helm Chart sends data to Sumo using the OpenTelemetry Protocol (OTLP), and therefore uses the [OTLP Source][otlp_source]. However, if you've chosen to use a [plain Http Source][http_source] by setting `sumologic.logs.source_type` to `http`, be aware that this source does not support client-side multiline parsing for logs in `text` format. You'll need to do multiline detection in the source itself. This can be set up in the Helm Chart configuration the following way:
```